### DISCUSSION TITLE
how to remove a library from the Arduino libraries list<title>

### DISCUSSION THREAD

**Discussion by peterclarke2** [community/community#128196] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi all, I have a DS3231 library for that real time clock. The author is Henning Karlson from Rinky Dink electronics. I am trying to install it in my Arduino libraries folder. It is being rejected because there is a folder there with the same name. That folder is from Wicket, Ayers, Wippler, Nth Widget. Can anyone tell me how I can remove the present DS3231 folder so I can use Henning's much better library? Tried everything but so far no luck. I managed to uninstall the Wicket folder but it is still causing this problem because it is still listed.

**Comment by yeryoong** [comment]
First, try navigating to your Arduino libraries folder manually and locate the DS3231 folder from Wicket, Ayers, Wippler, Nth Widget. Once you've found it, try deleting it directly from there. If that doesn't work, you might want to check if there are any other instances of the DS3231 library lingering in your Arduino IDE's library manager or any other related settings. Removing it from there could help resolve the conflict. If all else fails, you could consider reaching out to the Arduino community forums or the developers of the conflicting library for further assistance. They might have encountered a similar issue and could provide some helpful insights or solutions.
Best of luck!

**Comment by peterclarke2** [comment]
Thanks for that. Yes there are quite a few with the same name but not exactly the same. Most have an extension like ‘RTC’. The one I am trying to use is just DS3231.
    Even though the Ayers one is not installed it is still in the libraries list. I see no way of removing it completely. I will reach out to the forum and the developers of the conflicting library as suggested. 

Sent from Mail for Windows
